docs/options.md

```diff
@@ -1,6 +1,26 @@
-# Cyclegan with mask labels
+# JoliGAN Models and Options
```

Comment:
Maybe we could generate the API doc directly from Python code, as yet another PR ?

Reply:
That's what I think too, I started to look at https://pypi.org/project/pydoc-markdown/